akka-kryo-serialization/src/main/scala-3/io/altoo/akka/serialization/kryo/serializer/scala/ScalaEnumSerializer.scala

```diff
@@ -0,0 +1,23 @@
+package io.altoo.akka.serialization.kryo.serializer.scala
+
+import com.esotericsoftware.kryo.{Kryo, Serializer}
+import com.esotericsoftware.kryo.io.{Input, Output}
+
+import scala.runtime.EnumValue
+
+class ScalaEnumSerializer[T <: EnumValue] extends Serializer[T]  {
```

Comment:
pls rename to ScalaEnumNameSerializer
to be consistent with https://github.com/EsotericSoftware/kryo/blob/master/src/com/esotericsoftware/kryo/serializers/EnumNameSerializer.java

Reply:
renamed